js/src/personal/amanda/LightDarkButton.tsx

```diff
@@ -0,0 +1,28 @@
+import {
+  ActionIcon,
+  Group,
+  useComputedColorScheme,
+  useMantineColorScheme,
+} from '@mantine/core'
+import { TbMoon, TbSun } from 'react-icons/tb'
+
+export function LightDarkButton() {
+  const { setColorScheme } = useMantineColorScheme()
+  const computedColorScheme = useComputedColorScheme('light')
+
+  return (
+    <Group justify="center">
+      <ActionIcon
+        onClick={() =>
+          setColorScheme(computedColorScheme === 'light' ? 'dark' : 'light')
```

Comment:
Feel free to ignore as this is more of a fyi, but there seems to just be a `toggleColorscheme` that we can use. Not sure why the doc suggested defining our own toggle logic w/ computed color scheme.

https://mantine.dev/theming/color-schemes/#use-mantine-color-scheme-hook